src/screens/Modals/AddEditModal.js

```diff
@@ -93,36 +87,21 @@ const AddEditModal = props => {
     const {balance, income, expense, transactionHistory} = props;
     let status = {balance, income, expense};
     let currentTransaction = {amount, desc, transactionType};
-    let isEditTransaction = `${recordID}`;
+    let isEditTransaction = props.modalType == 'Edit';
```

Comment:
Always try to use strict equality(`===`)

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality